File: LicentaUI/Models/AppointmentModel.cs
﻿using System;

namespace LicentaUI.Models
{
    public class AppointmentModel
    {
        public string IdAppointment { get; set; }

        public string Name { get; set; }

        public DateTime TimeAppointment { get; set; }

        public string IdBook { get; set; }

        public string TitleBook { get; set; }

        public string TitleFilm { get; set; }

        public string TitleSeries { get; set; }

        public string IdFilm { get; set; }

        public string IdSeries { get; set; }

        public string IdGroup { get; set; }

        public string Location { get; set; }
    }
}

File: LicentaAngular/src/app/models/appointment-model.ts
export interface AppointmentModel {
    idAppointment: string;
    name: string;
    timeAppointment: string;
    idBook: string;
    idFilm: string;
    idSeries: string;
    titleBook: string;
    titleFilm: string;
    titleSeries: string;
    idGroup: string;
    groupName: string;
    location: string;
}

export interface CreateAppointmentModel {
    name: string;
    timeAppointment: string;
    idBook: string;
    idFilm: string;
    idSeries: string;
    idGroup: string;
    location: string;
}
